lib/registries/addon/drafts/draft/route.ts

```diff
@@ -27,6 +27,7 @@ export default class DraftRegistrationRoute extends Route.extend({
                 { adapterOptions: { include: 'branched_from' } },
             );
             const node = yield draftRegistration.branchedFrom;
+            yield node.loadRelatedCount('children');
```

Comment:
This could be combined with the request for `branchedFrom` one line up:
```ts
const nodeId = draftRegistration.belongsTo('branchedFrom').id();
const node = yield this.store.findRecord('node', nodeId, {
  adapterOptions: {
    query: {
      related_counts: 'children',
    },
  },
);
```
(or something like that)